README.md

```diff
@@ -1 +1,102 @@
+  
+  {'anforande_id': ['46102c9a-e312-ea11-912c-901b0e9b71a8'], 'dok_datum': ['2019-11-29'], 'talare': ['Ali Esbati (V)'], 'parti': ['V'], 'protokoll_url_www': ['http://www.riksdagen.se/sv/Dokument-Lagar/Kammaren/Protokoll/Riksdagens-snabbprotokoll_H70941/#anf167'], 'dok_titel': ['Protokoll 2019/20:41 Fredagen den 29 november'], 'intressent_id': ['014386615025']}, 
+  
+  {'anforande_id': ['45102c9a-e312-ea11-912c-901b0e9b71a8'], 'dok_datum': ['2019-11-29'], 'talare': ['Arbetsmarknadsministern Eva Nordmark (S)'], 'parti': ['S'], 'protokoll_url_www': ['http://www.riksdagen.se/sv/Dokument-Lagar/Kammaren/Protokoll/Riksdagens-snabbprotokoll_H70941/#anf166'], 'dok_titel': ['Protokoll 2019/20:41 Fredagen den 29 november'], 'intressent_id': ['0661583406713']}, 
+  
+  {'anforande_id': ['44102c9a-e312-ea11-912c-901b0e9b71a8'], 'dok_datum': ['2019-11-29'], 'talare': ['Ali Esbati (V)'], 'parti': ['V'], 'protokoll_url_www': ['http://www.riksdagen.se/sv/Dokument-Lagar/Kammaren/Protokoll/Riksdagens-snabbprotokoll_H70941/#anf165'], 'dok_titel': ['Protokoll 2019/20:41 Fredagen den 29 november'], 'intressent_id': ['014386615025']}, 
+  
+  {'anforande_id': ['43102c9a-e312-ea11-912c-901b0e9b71a8'], 'dok_datum': ['2019-11-29'], 'talare': ['Arbetsmarknadsministern Eva Nordmark (S)'], 'parti': ['S'], 'protokoll_url_www': ['http://www.riksdagen.se/sv/Dokument-Lagar/Kammaren/Protokoll/Riksdagens-snabbprotokoll_H70941/#anf164'], 'dok_titel': ['Protokoll 2019/20:41 Fredagen den 29 november'], 'intressent_id': ['0661583406713']}, 
+  
+  {'anforande_id': ['42102c9a-e312-ea11-912c-901b0e9b71a8'], 'dok_datum': ['2019-11-29'], 'talare': ['Infrastrukturministern Tomas Eneroth (S)'], 'parti': ['S'], 'protokoll_url_www': ['http://www.riksdagen.se/sv/Dokument-Lagar/Kammaren/Protokoll/Riksdagens-snabbprotokoll_H70941/#anf163'], 'dok_titel': ['Protokoll 2019/20:41 Fredagen den 29 november'], 'intressent_id': ['0284192765516']}, 
+  
+  {'anforande_id': ['41102c9a-e312-ea11-912c-901b0e9b71a8'], 'dok_datum': ['2019-11-29'], 'talare': ['Jens Holm (V)'], 'parti': ['V'], 'protokoll_url_www': ['http://www.riksdagen.se/sv/Dokument-Lagar/Kammaren/Protokoll/Riksdagens-snabbprotokoll_H70941/#anf162'], 'dok_titel': ['Protokoll 2019/20:41 Fredagen den 29 november'], 'intressent_id': ['0216534495014']}, 
+  
+  {'anforande_id': ['40102c9a-e312-ea11-912c-901b0e9b71a8'], 'dok_datum': ['2019-11-29'], 'talare': ['Infrastrukturministern Tomas Eneroth (S)'], 'parti': ['S'], 'protokoll_url_www': ['http://www.riksdagen.se/sv/Dokument-Lagar/Kammaren/Protokoll/Riksdagens-snabbprotokoll_H70941/#anf161'], 'dok_titel': ['Protokoll 2019/20:41 Fredagen den 29 november'], 'intressent_id': ['0284192765516']}, 
+```
+
+members
```

Comment:
You don't need to give so many examples, for simplicity, just show 2 or max 3 results from the members api